unit/models/card.py

```diff
@@ -139,6 +139,22 @@ def from_json_api(_id, _type, attributes, relationships):
         return BusinessVirtualCreditCardDTO(BusinessVirtualCardDTO.from_json_api(_id, _type, attributes, relationships))
 
 
+class CreateCardLimits(object):
```

Comment:
We actually have that class already, **CardLevelLimits**.
please use the one that exists. You will see that CardLevelLimits is a subclass of UnitDTO and has a to_dict method that returns a Dict like you created.
We don't need to call to_dict explicitly anymore, it executes automaticlly as part of the Encoder